src/stash-list/bookmark.vue

```diff
@@ -20,7 +20,16 @@
      @auxclick.middle.exact.prevent.stop="closeOrHideOrOpen">
      {{bookmark.title}}
   </a>
+  <input
+    ref="newtitle"
+    v-show="isEditable"
```

Comment:
Using `v-if` and `v-else` (for `<input>` and `<a>`) is probably better for performance here since we expect to edit only one bookmark at a time. If you use `v-show`, one `<input>` element will be created up front for every bookmark, which means page loading (and bookmark updates) will take longer.

Reply:
good call, done